src/test/regress/sql/schemas_create.sql

```diff
@@ -0,0 +1,9 @@
+CREATE SCHEMA test_schema_1;
+CREATE SCHEMA IF NOT EXISTS test_schema_2;
+CREATE SCHEMA test_schema_3 CREATE TABLE test_table(a INT PRIMARY KEY);
+
+SELECT create_distributed_table('test_schema_3.test_table','a');
+INSERT INTO test_schema_3.test_table VALUES (1), (2);
+
```

Comment:
We may also add the following tests:
- `ALTER TABLE test_schema_3.test_table SET SCHEMA test_schema_4`
- `ALTER TABLE test_schema_3 RENAME TO test_schema_3_renamed`